docs/src/terminology.md

```diff
@@ -194,6 +214,10 @@ The public key of the [account](terminology.md#account) containing a [program](t
 
 A stack of proofs, each which proves that some data existed before the proof was created and that a precise duration of time passed before the previous proof. Like a [VDF](terminology.md#verifiable-delay-function), a Proof of History can be verified in less time than it took to produce.
 
+## proof of stake 
+
+An economic mechanism used to: align validator incentives with those of the cluster itself, punish malicious behaviour through slashing, and provide a rewards scheme for both consistent validation of the network as well as token delegation by end users towards validators.
```

Comment:
I don't love this, but don't have a suggestion. I'd rather leave this out for now.